internal/services/controller/controller.go

```diff
@@ -354,7 +250,11 @@ func (c *Controller) collectInitialSnapshot(ctx context.Context) error {
 	// Collect initial state from cached informers and push to deltas queue.
 	for objType, informer := range c.informers {
 		for _, item := range informer.GetStore().List() {
-			c.genericHandler(c.log, objType, eventAdd, item)
+			handler, found := c.handlers[objType]
+			if !found {
+				return fmt.Errorf("handler for type %v not found", objType.String())
```

Comment:
Yes, this would indicate that no local testing was done, both of the maps are initialized side by side.

I also updated this spot in upcoming PR, this lookup will no longer exist.